parameter_files/fates_params_default.cdl

```diff
@@ -1098,12 +1220,16 @@ data:
 
  fates_cohort_fusion_tol = 0.08 ;
```

Comment:
do we need to change the name of this to fates_cohort_size_fusion_tol to distinguish from the new age variable?

Reply:
yeah, @ckoven , I like that
"fates_cohort_size_fusion_tol"